src/chat.h

```diff
@@ -113,6 +113,13 @@ class ChatBuffer
 	// Scroll to top of buffer (oldest)
 	void scrollTop();
 
+	// Functions for keeping track of whether the lines were modified by any
+	// preceding operations
+	// If they were not changed, getLineCount() and getLine() output the same as
+	// before
+	bool areLinesChanged() { return m_lines_changed; }
+	void markLinesUnchanged() { m_lines_changed = false; }
```

Comment:
naming suggestion
```suggestion
	bool getModified() const { return m_lines_changed; }
	void resetModified() { m_lines_changed = false; }
```

Reply:
I've not called it "modified" because there are also other things that can be modified, ie. the scrolling and the formatted text (those are not interesting for us). Will do the const though.

Reply:
Okay.